### PR TITLE
testing get_next_mapping_aks.sh

### PR DESCRIPTION
**Pull request by temitope777**

### CODE REVIEW

**Comment by MylesJarvis** [comment]
Tested - I can sign into DfE Sign In and act as a school. Therefore think this is all working as intended.

File: terraform/aks/variables.tf [hunk 1]
```diff
@@ -141,6 +141,13 @@ variable "review_url_redis_name" {
   default     = null
   description = "The name of the secret storing review redis url"
 }
+
+
+variable "dfe_signin_base_url" {
```
Comment (saliceti) [inline terraform/aks/variables.tf:146]: I believe this can be removed now

File: terraform/aks/variables.tf [hunk 2]
```diff
@@ -149,9 +156,12 @@ variable "dsi_hostname" {
   description = "The static hostname for DFE sign-in "
   default = ""
 }
+variable "create_dsi_ingress" {
+  description = "Optional addicional ingress for DSI hostname when front door is not used"
```
Comment (saliceti) [inline terraform/aks/variables.tf:160]: Typo: additional

File: terraform/aks/config/staging.tfvars.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
     "namespace": "git-test",
     "environment": "staging",
     "infra_key_vault_name": "s189t01-gse-stg-inf-kv",
+    "key_vault_name": "s189t01-gse-stg-app-kv",
```
Comment (saliceti) [inline terraform/aks/config/staging.tfvars.json:6]: I believe key_vault_name can be removed from all the tfvars files

Comment (temitope777) [reply terraform/aks/config/staging.tfvars.json:6]: needed by review tfvars though. others removed

**Review by saliceti — APPROVED** [review]

**Pull request merged**